contracts/exchange-libs/contracts/src/LibEIP712.sol

```diff
@@ -21,54 +21,65 @@ pragma solidity ^0.5.5;
 
 contract LibEIP712 {
 
-    // EIP191 header for EIP712 prefix
-    string constant internal EIP191_HEADER = "\x19\x01";
-
-    // EIP712 Domain Name value
-    string constant internal EIP712_DOMAIN_NAME = "0x Protocol";
-
-    // EIP712 Domain Version value
-    string constant internal EIP712_DOMAIN_VERSION = "2";
-
     // Hash of the EIP712 Domain Separator Schema
     bytes32 constant internal EIP712_DOMAIN_SEPARATOR_SCHEMA_HASH = keccak256(abi.encodePacked(
         "EIP712Domain(",
         "string name,",
         "string version,",
+        "uint256 chainId,",
```

Comment:
Nit: should this be `chainId` or `networkId`? Feels like we should just stick to one, and `networkId` is already used in the standard RPC methods.

Reply:
Discussed offline, these are actually 2 different things.